src/Bitmovin/api/container/JobContainer.php

```diff
@@ -96,6 +97,12 @@ public function getOutputPath()
             $prefix = $this->addLeadingSlash($prefix);
             return $this->addTrailingSlash($prefix);
         }
+        else if ($output instanceof S3Output)
+        {
+            $prefix = $this->stripSlashes($output->prefix);
+            $prefix = $this->addLeadingSlash($prefix);
```

Comment:
Leading slash is not needed for S3Output. You could merge this block the GCE Output.